spec.md

```diff
@@ -387,6 +387,8 @@ Uploads are started with a POST request which returns a url that can be used to
 
 The `Location` header will be used to communicate the upload location after each request.
 While it won't change in the this specification, clients SHOULD use the most recent value returned by the API.
+Note that the `Location` header value returned may either be absolute or relative as described in
```

Comment:
The "Note that" doesn't seem necessary, try "The `Location` header value MAY be either an absolute or relative URI as described in"

Reply:
Pushed an update per comments. MUST instead of MAY makes more sense here?